numba/cuda/tests/cudadrv/test_context_stack.py

```diff
@@ -21,5 +24,19 @@ def test_gpus_iter(self):
         self.assertGreater(len(gpulist), 0)
 
 
+class TestContextAPI(unittest.TestCase):
```

Comment:
Can you add `SerialMixin` to this class?  It is necessary to run correctly when the test runner is given the `-m` multiprocessing flag.